wellregistry/registry/migrations/0013_registry_update_well_depth.py

```diff
@@ -0,0 +1,85 @@
+# The below suppresses pylint message: Module name "*" doesn't conform to snake_case naming style
+# pylint: disable-msg=C0103
+# Enable check for the rest of the file
+# pylint: enable-msg=C0103
+"""
+# Updating fields including well depth which did not allow for well depths over 100)
```

Comment:
I did have to make a change to the well depth field but most of these are left over from the PR for validations. Recommend that make runmigrations is run before making a PR to make sure that we don't get migration warnings about missing migrations.